yt/frontends/nc4_cm1/tests/test_outputs.py

```diff
@@ -0,0 +1,69 @@
+from yt.frontends.nc4_cm1.api import CM1Dataset
+from yt.testing import assert_equal, requires_file, units_override_check
+from yt.utilities.answer_testing.framework import (
+    FieldValuesTest,
+    GridValuesTest,
+    can_run_ds,
+    data_dir_load,
+    requires_ds,
+    small_patch_amr,
+)
+
+_fields = ("thrhopert", "zvort")
+cm1sim = "cm1_tornado_lofs/nc4_cm1_lofs_tornado_test.nc"
+
+
```

Comment:
Should an answer test be added here?

Reply:
`test_cm1_mesh_fields` is an answer test, ya?